generators/app/templates/src/redux/auth/actions.ejs

```diff
@@ -1,29 +1,33 @@
+import { ApiOkResponse } from 'apisauce';
+import { Dispatch } from 'react';
 import { SwitchActions } from 'react-navigation';
 import { createTypes, completeTypes, withPostSuccess } from 'redux-recompose';
-import * as AuthService from '@services/AuthService';
 import Routes from '@constants/routes';
+import { CurrentUser } from '@interfaces/authInterfaces';
+import * as AuthService from '@services/AuthService';
 
 export const actions = createTypes(completeTypes(['LOGIN'], ['AUTH_INIT', 'LOGOUT']), '@@AUTH');
 
-const loginTarget = 'currentUser';
+const TARGETS = {
+  CURRENT_USER: 'currentUser'
+};
 
 export const actionCreators = {
-  init: user => ({ type: actions.AUTH_INIT, target: loginTarget, payload: user }),
-  login: authData => ({
+  init: (user: CurrentUser) => ({ type: actions.AUTH_INIT, target: TARGETS.CURRENT_USER, payload: user }),
+  login: () => ({
     type: actions.LOGIN,
-    target: loginTarget,
+    target: TARGETS.CURRENT_USER,
     service: AuthService.login,
-    payload: authData,
     injections: [
-      withPostSuccess((dispatch, response) => {
+      withPostSuccess((dispatch: Dispatch<any>, response: ApiOkResponse<CurrentUser>) => {
```

Comment:
If instead of any you use the `Action` you defined in `interfaces` ? Or.. I'm not sure maybe you can pass it something with the types of the Actions that will be dispatched in that `withPostSuccess` statement. For example the type in this case will be a `JUMP_TO` action. 
But really I'm not sure I didn't find much info of how to type this but I think we should research a bit more to avoid people do this in future projects

Reply:
Done